### PR TITLE
ReaderDogEar: Enforce a minimum size, too

### LINKED ISSUE #7364: FR: Minimum (or fixed) size for dogear mark
I like reading with minimalistic margins (the device screen is small enough to waste it with margins). Unfortunately, that makes the "dogear" mark for bookmarks really tiny and hard to see.

**Describe the solution you'd like**
I'd like a setting for minimum dogear size, or a fixed size, independent of the margins.

I use bookmarks seldom enough that possible overlapping with the text is not a concern for me, but for robustness, maybe the dogear mark could have a partial opacity, or invert the text underneath?

**Additional context**

Default bookmark with 5-size margins
Screenshot_20210228_112052

15-size boommark with 5-size margins (mockup)
Screenshot_20210228_112754

### PR DESCRIPTION
**Pull request by NiLuJe**
Also, update the actual ReaderDogEar variant of the icon to use a semi-transparent fill for the bottom-left corner.

Fix #7364

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/7369)

### CODE REVIEW

**Comment by NiLuJe** [comment]
Haven't checked how the min value looks on truly high-dpi devices (only checked on the H2O, not the Forma or a phone), so, open to more testing ;).

**Comment by Frenzie** [comment]
Meh, 260 vs 300 isn't super meaningful. 600 is though ;)

**Comment by NiLuJe** [comment]
Yeah, i think it ended up being hilariously tiny on my phone, but, I don't really use bookmarks (or my phone for reading, for that matter ;p) so it never really bothered me that much, but it is a low hanging fruit ;).

File: resources/icons/mdlight/dogear.alpha.svg [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+
+
+
+	
+
+		
+	</defs>
+
```
Comment (poire-z) [inline resources/icons/mdlight/dogear.alpha.svg:67]: Signing your art with blank lines ? :)

Comment (NiLuJe) [reply resources/icons/mdlight/dogear.alpha.svg:67]: Inkscape doing wierd Inkscape things, I guess ^^.

**Review by poire-z — APPROVED** [review]
Not tested but feels ok.

**Pull request merged**